src/jit/codegenxarch.cpp

```diff
@@ -1855,6 +1855,13 @@ CodeGen::genCodeForTreeNode(GenTreePtr treeNode)
                         type0 = TYP_REF;
                         firstIntUsed = true;
                         break;
+                    case SystemVClassificationTypeIntegerByReference:
```

Comment:
Clearly this is code duplication and hence requiring to make the same fix at two places.  We should take this opportunity to abstract this into a routine.